README.md

```diff
@@ -164,6 +164,41 @@ $ LD_LIBRARY_PATH=./build ./build/simple_test
 Copy `OpenCL.dll` into a system location or alongside the application executable
 you want to run with clvk.
 
+## Tuning clvk performances
+
+clvk can be tuned to improve performances. While we try to have the default
+parameters set at their best values for each platform, they can sometimes be
+changed for specific applications. One of the best way to know whether something
+can be improved is to use traces to understand what should be changed.
+
+### Group size
+
+clvk implementation is grouping commands waiting for a call to clFlush or any
+blocking calls (clFinish, clWaitForEvents, etc.) to flush group.
+
+clvk has 2 variables to allow to flush group earlier depending on the number of
+commands in the group:
+   - `CLVK_MAX_CMD_GROUP_SIZE`
+   - `CLVK_MAX_FIRST_CMD_GROUP_SIZE`
+
+`CLVK_MAX_FIRST_CMD_GROUP_SIZE` is used if there is no group to be process or
+being processed in the queue.
+Otherwise `CLVK_MAX_CMD_GROUP_SIZE` is used.
```

Comment:
I think I'd remove those three lines. The documentation for the individual variables below covers it.